engine/server/engine/centralized_logs/client_implementations/persistent_volume/log_file_manager/log_file_manager.go

```diff
@@ -136,7 +136,9 @@ func (manager *LogFileManager) RemoveLogsBeyondRetentionPeriod() {
 }
 
 func (manager *LogFileManager) RemoveAllLogs() error {
-	if err := manager.filesystem.RemoveAll(volume_consts.LogsStorageDirpath); err != nil {
+	// only removes logs for this year because Docker prevents all logs from base logs storage file path
```

Comment:
doing only for this year is a hack to get rid of all existing logs storage bc removing logs at the base path isn't allowed by Docker (for some reason) - but it should do the trick for clearing up all historical logs that are taking up current user log storage